test/unit-fortran/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+cmake_minimum_required(VERSION 3.18)
```

Comment:
Now that there is a `unit-fortran` directory, can we rename the current `unit` directory to `unit-python`, to help make it clear what the difference is?  I should note that this change will probably require an update to the `python_unit_tests.yml` workflow file along with changes to the `run_unit_tests.sh` testing shell script (which should probably be re-named to `run_python_unit_tests.sh` as well).

Reply:
Actually that was going to be my original suggestion, it just might take a little more work to make sure that the python unit tests continue working as they have some assumed paths.  However if you are willing to give it a try in this PR then I totally agree that your suggested directory structure is a better one.  I'm also happy to help if you run into any weird test failures!

Reply:
Moved!